index.js

```diff
@@ -37,8 +37,11 @@ var merge = function(a, b) {
 
```

Comment:
Should this be squashed with previous commit or should they be left separate for problem demonstration?